rave-compiler/src/test/java/com/uber/rave/model/IntRangeTestModel.java

```diff
@@ -14,31 +14,31 @@
  */
 @Validated(factory = MyFactory.class)
 public class IntRangeTestModel {
-    private static final long FROM = -15L;
-    private static final long TO = 1000L;
+    private static final int FROM = -15;
```

Comment:
I think we can actually leave these since `IntRange` defaults to longs. Thoughts?

Reply:
I've posted in ASG if a similar change is going to be made to IntRange lets see what Ian says.

Reply:
I actually like this better that supporting longs and I suspect IntRange will eventually move to the same thing per the problems described with IntDef here:

https://android.googlesource.com/platform/frameworks/support/+/df64f2ab05e32519d95adc1625cdd840f650956c